README.md

```diff
@@ -56,6 +56,20 @@ import pyfuncol
 # {"a": 1, "b": 4, "c": 9}
 ```
 
+pyfuncol provides operations leveraging memoization to improve performance (for now `pure_map` and `pure_flat_map`). BEWARE: these versions work only
```

Comment:
Actually, could you update the README to do the same as here but for parallel operations? So we have 3 blocks of code, one for the normal ops, one for the parallel ones and one for the pure

Reply:
Yep, sure :)